src/modules/UI/scenes/SendConfirmation/SendConfirmation.ui.js

```diff
@@ -308,10 +309,11 @@ export class SendConfirmation extends Component<Props, State> {
       const fiatFeeSymbol = secondaryInfo.displayDenomination.symbol ? secondaryInfo.displayDenomination.symbol : ''
       const exchangeConvertor = convertNativeToExchange(this.props.parentExchangeDenomination.multiplier)
       const cryptoFeeExchangeAmount = exchangeConvertor(parentNetworkFee)
-      const fiatFeeAmount = this.props.currencyConverter.convertCurrency(
+      const fiatFeeAmount = convertCurrencyFromExchangeRates(
+        exchangeRates,
         this.props.parentExchangeDenomination.name,
         secondaryInfo.exchangeCurrencyCode,
-        cryptoFeeExchangeAmount
+        parseFloat(cryptoFeeExchangeAmount)
       )
```

Comment:
We will need `cryptoFeeExchangeAmount` as a number so that we can multiply it by the exchange rate